src/scenes/Scene.js

```diff
@@ -49,18 +49,6 @@ Scene.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 	},
 
-	toJSON: function ( meta ) {
-
-		var data = Object3D.prototype.toJSON.call( this, meta );
-
-		if ( this.background !== null ) data.object.background = this.background.toJSON( meta );
```

Comment:
We have to move this section to `Object3D.toJSON()` since the `meta` object does not contain `textures` and `images` properties when this overwritten method is called with a scene just containing a background and untextured objects.